packages/commands/serve/src/index.ts

```diff
@@ -61,20 +62,19 @@ export default createCommand<
       try {
         fake(schema);
 
-        const server = createServer({
-          schema,
-          port,
-          cors: true,
-          logging: false,
+        const yoga = createYoga({ schema, cors: true, logging: false });
```

Comment:
@dotansimha Did you review this also right? because the old package version was deprecated so i change it to Yoga.